src/pudl/helpers.py

```diff
@@ -983,7 +983,8 @@ def convert_to_date(
         year_col: the name of the year column in the original table.
         month_col: the name of the month column in the original table.
         day_col: the name of the day column in the original table.
-        month_value: generated month if no month exists.
+        month_value: generated month if no month exists or if the month
+            value is NA.
```

Comment:
I updated this function because there was a month column but it contained NA values for the years where it was annual vs. monthly. I wanted to fill those in with 1, but the previous `convert_to_date` function only fills in the month if there is no month column at all, which there is. So I edited it to fill in the month value if the month column contains NA values. 

I did not check whether this has any downstream effects in other tables, but my guess is that it doesn't because most tables are not combining annual and monthly data and therefore don't have NA values in the month column.